engine/src/main/java/org/terasology/rendering/nui/layouts/CardLayout.java

```diff
@@ -26,20 +26,43 @@
 import java.util.Iterator;
 import java.util.Map;
 
+/**
+ * A layout that allows for a single widget to be displayed among a list of stored widgets
+ */
 public class CardLayout extends CoreLayout<LayoutHint> {
+    /**
+     * The id of the currently displayed widget
+     */
     @LayoutConfig
     private String defaultCard;
```

Comment:
That been said: wouldn't it make more sense to rename the variable to currentlyDisplayedWidget or currentlyDisplayedCard, warping ahead toward self-documenting code? :smile: